src/renderers/webgl/WebGLLights.js

```diff
@@ -199,10 +218,13 @@ function WebGLLights() {
 					uniforms.shadowRadius = shadow.radius;
 					uniforms.shadowMapSize = shadow.mapSize;
 
+					state.directionalShadowMap[ directionalLength ] = shadowMap;
+					state.directionalShadowMatrix[ directionalLength ] = light.shadow.matrix;
+
+					++ numDirectionalShadows;
```

Comment:
Can you please write `numDirectionalShadows ++` for consistency reasons 😇

Reply:
Done.